coding/move_to_front.hpp

```diff
@@ -3,6 +3,7 @@
 #include <array>
 #include <cstdint>
 #include <limits>
+#include <cstddef>
```

Comment:
sort, please

Reply:
Do you mean place `#include <cstddef>` before `#include <cstdint>` string?

Reply:
Yes.